api/app/controllers/sessions_controller.rb

```diff
@@ -65,4 +67,23 @@ def set_session
   def session_params
     params.require(:session).permit(:name, :description, :location, :meeting_link, :start_time, :end_time, :group_id)
   end
+
+  def authorize_group_member!
+    group = @session ? @session.group : Group.find(session_params[:group_id])
+
+    member = Member.find_by(user: current_user, group:)
```

Comment:
Podríamos tener un `set_member` como tenemos el `set_session` acá, que te parece?
Así te evitás hacer esto acá y también en el `create`. Vas a poder acceder al member desde `@member` después.

Reply:
Este me lo llevo meter en un pr a parte para no trancar. Justo ando por salir y hasta la noche no vuelvo, y quiero meterlo con los test correspondientes para el not found